app/controllers/carts_controller.rb

```diff
@@ -16,6 +18,13 @@ def index
     @items = @cart.items
   end
 
+  def update
+    @cart.update_item(cart_params)
```

Comment:
@kheppenstall I did the permit just to simplify the passed hash. I'm fine with just passing params, as I see it might be an unnecessary use of strong params.